src/openrct2-ui/interface/Dropdown.h

```diff
@@ -37,29 +37,33 @@ namespace Dropdown
     void SetImage(int32_t index, ImageId image);
 } // namespace Dropdown
```

Comment:
Shouldn't this have been inside the `Ui` namespace too?

Reply:
Yes it should but i was getting in a situation where i had to keep changing more and more and more. So thought it best to just do it in multiple parts